src/examples/forms/checkbox/Default.tsx

```diff
@@ -17,21 +17,24 @@ const Example = () => {
   return (
     <Row justifyContent="center">
       <Col size="auto">
-        <Field>
-          <Checkbox checked={pest} onChange={() => setPest(!pest)}>
-            <Label>Pest resistant</Label>
-          </Checkbox>
-        </Field>
-        <Field>
-          <Checkbox checked={light} onChange={() => setLight(!light)}>
-            <Label>Needs direct light</Label>
-          </Checkbox>
-        </Field>
-        <Field>
-          <Checkbox checked={drought} onChange={() => setDrought(!drought)}>
-            <Label>Drought-tolerant</Label>
-          </Checkbox>
-        </Field>
+        <Fieldset>
+          <Fieldset.Legend>plant preference</Fieldset.Legend>
```

Comment:
```suggestion
          <Fieldset.Legend>Plant preference</Fieldset.Legend>
```

Must be sentence case per [content documentation](https://garden.zendesk.com/content/grammar#capitalization).